rules/windows/lateral_movement_defense_evasion_lanman_nullsessionpipe_modification.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/03/22"
+maturity = "production"
+updated_date = "2021/03/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies NullSessionPipe registry modifications that specify which pipes that can be accessed anonymously. This could
+be indicative of adversary lateral movement preparation by making the added pipe available to everyone
+(untrusted/anonymous integrity level).
```

Comment:
```suggestion
Identifies NullSessionPipe registry modifications that specify which pipes can be accessed anonymously. This could
be indicative of adversary lateral movement preparation by making the added pipe available to everyone.
```